docs/5.x/quickstart.md

```diff
@@ -1,231 +1,373 @@
+### Step 4: Building the Application Bundle
 
-While the resulting Application Bundle can be used as is, Gravity also allows you
-to publish it in your Ops Center for distribution. Publishing it in the Ops Center also allows
-you to see how many instances of the application are running and even connect to them remotely.
+Before we build our first Application Bundle,   let's make sure [Helm](https://helm.sh/) is properly 
+initialized and [helm-template](https://github.com/technosophos/helm-template) plugin is installed:
 
-To publish an application, you have to connect to the Ops Center first.
-You should already have an account setup with Gravitational - usually available
-on `https://[yourdomain].gravitational.io`
+```bash
+$ helm init --client-only
+$ helm plugin install https://github.com/technosophos/helm-template
+```
 
-To log in to to the Ops Center using the CLI:
+Now you can build the Application Bundle which will consist of a Kubernetes
+cluster with Mattermost pre-installed inside:
 
 ```bsh
-$ tele login -o [yourdomain].gravitational.io
-```
+$ tele build -o mattermost.tar mattermost/resources/app.yaml
 
-!!! tip "Note":
-    If you are using a guest environment, like Vagrant on macOS, you may see an error
-    when attempting due to network restrictions. If so, first get a temporary key from
-    your host environment with the following command:
+# the output:
+* [1/6] Selecting application runtime
+	Will use latest runtime version 5.2.4
+* [2/6] Downloading dependencies from s3://hub.gravitational.io
+	Still downloading dependencies from s3://hub.gravitational.io (10 seconds elapsed)
+	Still downloading dependencies from s3://hub.gravitational.io (20 seconds elapsed)
+* [3/6] Embedding application container images
+	Detected application manifest app.yaml
+	Detected resource file clusterDeprovision.yaml
+	Detected resource file clusterProvision.yaml
+	Detected resource file install.yaml
+	Detected resource file nodesDeprovision.yaml
+	Detected resource file nodesProvision.yaml
+	Detected Helm chart charts/mattermost
+	Using local image quay.io/gravitational/debian-tall:0.0.1
+	Using local image quay.io/gravitational/debian-tall:0.0.1
+	Using local image quay.io/gravitational/provisioner:ci.82
+	Using local image mattermost-worker:2.2.0
+	Using local image postgres:9.4.4
+	Vendored image gravitational/debian-tall:0.0.1
+	Vendored image gravitational/provisioner:ci.82
+	Still embedding application container images (10 seconds elapsed)
+	Vendored image mattermost-worker:2.2.0
+	Vendored image postgres:9.4.4
+* [4/6] Using runtime version 5.2.4
+	Still using runtime version 5.2.4 (10 seconds elapsed)
+* [5/6] Generating the cluster snapshot
+* [6/6] Saving the snapshot as mattermost.tar
+* [6/6] Build completed in 2 minutes 
+```
 
-```bsh
-$ tele keys new
+Let's review what just happened. `tele build` did the following:
+
+* Downloaded Kubernetes binaries and Gravity tooling from `s3://hub.gravitational.io`. The application "runtime".
```

Comment:
`The application "runtime".` sentence seems a bit out of place. If it's supposed to clarify the previous sentence, I would either write it after a dash (Downloaded blah-blah - the application "runtime") or inside parentheses.